css/properties/flex.json

```diff
@@ -46,36 +46,33 @@
                 "version_added": "20",
                 "notes": [
                   "Since Firefox 28, multi-line flexbox is supported.",
-                  "Before Firefox 32, Firefox wasn't able to animate values starting or stopping at <code>0</code>."
+                  "Before Firefox 32, Firefox wasn't able to animate values starting or stopping at <code>0</code>.",
+                  "Until Firefox 61, flex items that are sized according to their content are sized using <a href='https://drafts.csswg.org/css-sizing-3/#column-sizing'>fit-content, not max-content</a>."
                 ]
               },
               {
-                "version_added": "20",
-                "version_removed": "61",
-                "notes": "Flex items that are sized according to their content are sized using <a href='https://drafts.csswg.org/css-sizing-3/#column-sizing'>fit-content, not max-content</a>."
-              },
-              {
-                "prefix": "-webkit-",
-                "version_added": "49"
-              },
-              {
-                "prefix": "-webkit-",
-                "version_added": "44",
+                "version_added": "18",
+                "version_removed": "20",
```

Comment:
Was this not removed in 28 after all? Would be nice to substantiate this change

Reply:
Took me a while to decipher what you were talking about in this one due to the semi-borked changelog, but this applies for all of the review comments.  The way that I see it is that even if the preference remained, there should be a `version_removed` indicating when the value needed to be set was made the default; or in other words, "the need to set the flag X to value Y was removed in this version Z".  At least, this is how I would interpret the data.

Alternatively, I'm happy to revert these pieces to reflect that the flags still existed!

Reply:
Sorry about the confusion on this. I found it lots easier to look at this by switching to the split diff (instead of unified).

Anyway, we don't actually have a policy on so-called "dead flags" (that is, preferences that exist but are turned on by default). I was supposed to make a decision about this in #3318 and document it—I'm bumping it up my to do list. Absent a policy, the consistent thing to do is to let the on-by-default compat statement supersede the off-by-default compat statement. Made up example:

Firefox adds a feature, behind a preference:

* Firefox
  * version_added: 23; has some preference

Later, the feature is turned on by default:

* Firefox
  * version_added: 25
  * version_added: 23; has some preference

It's OK if support statements overlap like this, at least for now. Presumably, the preference is removed someday and it looks like this:

* Firefox
  * version_added: 25
  * version_added: 23; version_removed: 31; has some preference

Reply:
Gotcha.  Alrighty then, in that case let me bring it back closer to how it was before!